src/main/resources/application.yaml

```diff
@@ -72,28 +72,6 @@ info:
     environment: ${PACKAGES_ENVIRONMENT:local}
     project: ${PACKAGES_PROJECT:coh}
     name: ${PACKAGES_NAME:cor}
-#    version: ${PACKAGES_VERSION:unknown}
-
-#auth:
-#  idam:
-#    client:
-#      baseUrl: ${IDAM_USER_BASE_URI:http://localhost:4501}
-#  provider:
-#    service:
-#      client:
-#        baseUrl: ${S2S_BASE_URI:http://localhost:4502}
-#
-#
-#authorization:
-#  idam-roles-whitelist: ${IDAM_ROLES_WHITELIST:solicitor}
-#  s2s-names-whitelist: ${S2S_NAMES_WHITELIST:sscs,divorce,ccd,em_gw}
-#
-info:
-  app:
-    environment: ${PACKAGES_ENVIRONMENT:local}
-    project: ${PACKAGES_PROJECT:coh}
-    name: ${PACKAGES_NAME:cor}
-#    version: ${PACKAGES_VERSION:unknown}
```

Comment:
Had to remove all of it as Spring Boot 2.0.2 doesn't like duplicated keys in configuration file.